### PR TITLE
Add UnityMeta Token (BEP20)

### PR DESCRIPTION
**Pull request by unitymetatoken**
Adding info for BEP20 token 'UnityMeta Token'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by unitymetatoken** [comment]
please review this one


and closed #24252 Pull Request

**Comment by Cryptocool1** [comment]
Submission looks good awaiting for fee payment

**Comment by unitymetatoken** [comment]
ok sir @Cryptocool1 

Please Check 2 workflows awaiting approval

**Comment by unitymetatoken** [comment]
Please Also check File

**Comment by unitymetatoken** [comment]
https://user-images.githubusercontent.com/120736197/223151881-010741c5-668c-4155-93c4-07b45826fdb2.png

Please Remove logo.png.png

and please also check

**Comment by unitymetatoken** [comment]
Hey Sir @Cryptocool1 , You want to ask any question

**Comment by unitymetatoken** [comment]
Thanks

File: blockchains/smartchain/assets/0xca861e289f04cB9C67fd6b87ca7EAFa59192f164/info.json.json [hunk 1]
```diff
@@ -1,11 +1,44 @@
...
- "name": "UnityMeta Token",
- "website": "https://unitymeta.finance/",
- "description": "UnityMeta Token is a company that aims to bring innovation and novelty to the Metaverse, utilities, nft, sports, gaminig, and cryptotrading sector through Blockchain technology.",
- "explorer": "https://bscscan.com/token/0xca861e289f04cb9c67fd6b87ca7eafa59192f164",
- "type": "BEP20",
- "symbol": "UMT",
- "decimals": 18,
- "status": "active",
- "id": "0xca861e289f04cB9C67fd6b87ca7EAFa59192f164"
-}
```
Comment (UTOPIYA-AI) [inline blockchains/smartchain/assets/0xca861e289f04cB9C67fd6b87ca7EAFa59192f164/info.json.json]: blockchains/smartchain/assets/0xca861e289f04cB9C67fd6b87ca7EAFa59192f164/info.json.json

**Pull request merged**